src/adhocracy_core/adhocracy_core/websockets/server.py

```diff
@@ -51,6 +51,13 @@ def subscribe(self, client: Hashable, resource: IResource) -> bool:
         if self.is_subscribed(client, resource):
             return False
         path = resource_path(resource)
+
+        # ensure that no private info leaks to the outside
+        if path.startswith('/principals/') or path.startswith('/catalogs/'):
+            logger.info('Unallowed subscribtion')
+            raise WebSocketError('forbidden', 'You are not allowed to '
+                                 'subscribe on {}'.format(path))
+
```

Comment:
> Yves told me there are some usability problem when websocket are disabled.

ok, but then please add a TODO refactoring  comment and exclude the lines from test coverage

> I would appreciate you work on it since you have a deeper understanding of the websocket server. Or we work together on it.

Ok i do it,  for my proposed workaround you don't need deeper understanding of the websocket server.  

As proper solution I propose we ditch the websocket server and add a ServerSendEvents endpoint  to the rest api (that does not bypass the our security mechanisms)